product_pack/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_product_pack_line_employee,product.pack.line (se module README on why sale user is used here),model_product_pack_line,base.group_user,1,1,1,1
+access_product_pack_line_all,product.pack.line,model_product_pack_line,,1,0,0,0
```

Comment:
This NEVER should be global. At least, it must be for base user. Portal and public users must be studied to check if we should put them or not (annotate it on the README).